CDP4Addin/Resources/RibbonXml/addinribbon.xml

```diff
@@ -8,4 +8,7 @@
     <button id="CDP4_SelectModelToOpen" label="Open" screentip="Select an Engineering Model to open" imageMso="ServerConnection" onAction="OnAction" getEnabled="GetEnabled" />
     <button id="CDP4_SelectModelToClose" label="Close" screentip="Select an Engineering Model to close" imageMso="FileDropSqlDatabase" onAction="OnAction" getEnabled="GetEnabled" />
   </group>
+  <group id="CDP4info" label="Info">
+    <button id="CDP4_Plugins" label="Plugins" screentip="Select Plugins" imageMso="ProductGroup" onAction="OnAction" getEnabled="GetEnabled" />
```

Comment:
Here for `imageMso` use `DatabaseAnalyzeTable` to set a new icon that I picked.

Reply:
Image has been added. However, the image does not mach the IME icon for plugins